app/models/user_notification.rb

```diff
@@ -0,0 +1,9 @@
+class UserNotification < ApplicationRecord
+  belongs_to :notification
+  belongs_to :user
+
+  scope :unseen, ->{where seen: false}
```

Comment:
seen co the doi dung integer duoc khong? dung enum. neu sau nay muon them status read nua se de